### PR TITLE
Removed facedetect support

### PR DESCRIPTION
**Pull request by brusch**
Perfect candidate for bundle functionality ...

### CODE REVIEW

**Comment by dvesh3** [comment]
@brusch Should we also add a migration for cleaning up face detection custom settings from existing assets? otherwise they will be persisted unnecessarily.

File: bundles/AdminBundle/translations/admin.en.yaml [hunk 1]
```diff
@@ -882,9 +882,6 @@ batch_remove_selected: 'Batch remove from selected'
 batch_remove_from: 'Remove data from all'
 batch_remove_selected_from: 'Remove data from selected'
 unit_tooltip: 'Alternative units tooltip'
-toggle_image_features_visibility: 'Toggle Image Feature Visibility'
-detect_image_features: 'Detect Image Features'
-remove_image_features: 'Remove Image Features'
```
Comment (martineiber) [inline bundles/AdminBundle/translations/admin.en.yaml:887]: This exists in all other translation files as well. We should probably remove it there too?

Comment (brusch) [reply bundles/AdminBundle/translations/admin.en.yaml:887]: Not necessary, they are auto-generated and will be cleared on the next import 😊

**Comment by brusch** [comment]
@dvesh3 I was thinking about that too, but since it can't be undone I thought it probably makes sense to keep the values in the database. Maybe we can think about a command that can be run manually? 🤔

**Comment by dvesh3** [comment]
@brusch yes, the command will do 👍 however it should be done on 10.6 in preparation for 11 and then we remove the command on v11? or we can introduce a generic command to cleanup any custom setting from the assets and still keep it?

**Comment by brusch** [comment]
@dvesh3 I'd create a generic command and keep it, could be useful for other stuff as well. 

So basically: 

`bin/console pimcore:assets:remove-custom-setting faceCoordinates`

WDYT?

**Comment by brusch** [comment]
@dvesh3 @martineiber just added the command + upgrade notes 😊

File: bundles/CoreBundle/src/Command/Asset/RemoveCustomSettingCommand.php [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    {
+        $this
+            ->setName('pimcore:assets:remove-custom-setting')
+            ->setDescription('Removes a custom setting from assets')
+            ->addArgument('name', InputOption::VALUE_REQUIRED, 'Name of the custom setting to remove')
+            ->addOption(
+                'id',
+                null,
+                InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY,
+                'only generate checksum for assets with this (IDs)'
```
Comment (dvesh3) [inline bundles/CoreBundle/src/Command/Asset/RemoveCustomSettingCommand.php]:
```suggestion
                'only remove custom-setting for assets with this (IDs)'
```

File: bundles/CoreBundle/src/Command/Asset/RemoveCustomSettingCommand.php [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            )
+            ->addOption(
+                'parent',
+                'p',
+                InputOption::VALUE_OPTIONAL,
+                'only generate checksum for assets in this folder (ID)'
```
Comment (dvesh3) [inline bundles/CoreBundle/src/Command/Asset/RemoveCustomSettingCommand.php]:
```suggestion
                'only remove custom-setting for assets in this folder (ID)'
```

File: bundles/CoreBundle/src/Command/Asset/RemoveCustomSettingCommand.php [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+            )
+            ->addOption(
+                'path-pattern',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'only generate checksum for the given regex pattern (path + filename), example:  ^/Sample.*urban.jpg$'
```
Comment (dvesh3) [inline bundles/CoreBundle/src/Command/Asset/RemoveCustomSettingCommand.php]:
```suggestion
                'only remove custom-setting for the given regex pattern (path + filename), example:  ^/Sample.*urban.jpg$'
```

**Review by martineiber — APPROVED** [review]
Nice! Works fine.

**Comment by dpfaffenbauer** [comment]
@brusch was it moved to a bundle then? or just removed?

**Comment by brusch** [comment]
@dpfaffenbauer just removed from our side, not sure how the demand is on this feature - don't think that much 🤔

**Comment by studio1gmbh-pimcore** [comment]
@brusch is an alternative planed from your side? For us, this was a very useful feature in combination with thumbnail cropping, especially in cms projects.

**Comment by brusch** [comment]
@studio1gmbh-pimcore not yet 😬 but there are already so many other impressive tools out there which can be probably integrated quite easily 😉

**Pull request merged**